### PR TITLE
feat(config): support config upsert

### PR DESCRIPTION
**Pull request by tundeaoni**
Adds support for config values upsert as described in this issue https://github.com/kosyfrances/rundeck-zabbix/issues/24

### CODE REVIEW

**Comment by kosyfrances** [comment]
@tundeaoni Does this PR also fix #23? Was wondering whether to close it instead.

File: cli/cmd/setup.go [hunk 1]
```diff
@@ -66,6 +78,32 @@ var setupCmd = &cobra.Command{
...
+	return result
+}
+
+// mask values retrieved from config
+//https://play.golang.org/p/E4k4zT9ATK
+func mask(value string) string {
+	if len(value) <= 4 {
+		return value
+	}
+	return strings.Repeat("*", len(value)-4) + value[len(value)-4:]
```
[the remark this answers is not in the document]
Comment (tundeaoni) [reply cli/cmd/setup.go]: obscuring sensitive values/secrets like api-tokens

File: cli/cmd/setup.go [hunk 2]
```diff
@@ -66,6 +78,32 @@ var setupCmd = &cobra.Command{
 	},
 }
 
+// gets value from config file loaded via viper
+func getConfigValueByKey(key string) string {
```
[the remark this answers is not in the document]
Comment (tundeaoni) [reply cli/cmd/setup.go]: I will try this out

[the remark this answers is not in the document]
Comment (tundeaoni) [reply cli/cmd/setup.go]: Thanks

[the remark this answers is not in the document]
Comment (tundeaoni) [reply cli/cmd/setup.go]: Will address in a subsequent PR

**Pull request merged**